MiniSim/MiniSim.swift

```diff
@@ -106,13 +112,21 @@ class MiniSim: NSObject {
             UserDefaults.Keys.enableiOSSimulators: true
         ])
     }
+   
+    private func configureMenuImages() {
```

Comment:
let's move this logic to the `MenuImage` enum: 

```swift
enum MenuImage: String, CaseIterable {
    case iphone = "iphone"
    case ipad = "ipad"
    case box = "box"
    
    var image: NSImage? {
        guard let itemImage = NSImage(systemSymbolName: self.rawValue, accessibilityDescription: self.rawValue) else {
            return nil
        }
        itemImage.size = NSSize(
            width:  (itemImage.size.width)  * 0.78,
            height: (itemImage.size.height) * 0.78)
        itemImage.isTemplate = true
        return itemImage
    }
}
```

Reply:
Then you can call `MenuImage.iphone.image`

Reply:
configureMenuImages is only ran once because the image gets smaller and smaller every single time the function is reran, NSImage remembers the size. Pardon me if I misunderstand, but this would supposedly assume that this variable would be called multiple times?

Reply:
Image was getting smaller and smaller because you was mutating the button image in this case each item would have its own image